wherehows-frontend/conf/routes

```diff
@@ -266,15 +266,25 @@ GET           /api/v1/jira/tickets/:managerId
 
 POST          /api/v1/tracking                                                  controllers.api.v1.Tracking.addTrackingEvent()
 
-GET           /api/v2/dataset/listNames                                         controllers.api.v2.Dataset.listNames()
+GET           /api/v2/platform/:platform/:prefix                                controllers.api.v2.Dataset.listSegments(platform: String, prefix: String)
```

Comment:
minor change request, can we use the plural form for the resource name `platforms` since this is the common approach for collections and `:platform` refers to one item within that collection. otherwise, looks good 👍